src/modules/getUserData.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const getUserData = async id => {
+  let headers = JSON.parse(localStorage.getItem("J-tockAuth-Storage"));
+  let response = await axios
+    .get(`/admin/users/${id}`,
+    {
+      headers: headers
+    })
+    .then(response => {
```

Comment:
why are you not using a `try, catch` block here instead?
keep it consistent

Reply:
fixed